### DISCUSSION TITLE
Which UF2 firmware file should be copied to the system processor?

### DISCUSSION THREAD

**Discussion by jds-designs** [gizmo-platform/gizmo#21] [Q&A] [answered]
Hello, I am new to the Raspberry Pi system, so please bear with me if I'm overlooking something simple. I am trying to load the firmware on my hub's Gizmo system processors, and I'm unsure which file from the firmware GitHub repository (https://github.com/gizmo-platform/firmware/releases/) to use.

The documentation/guide (https://gizmoplatform.dev/startup/firmware.html) states, "Obtain the latest firmware release from [here](https://github.com/gizmo-platform/firmware/releases/). You need the uf2 file from the release artifacts located at the bottom of the card."

To my understanding, this is referring to a singular file. But the latest release contains multiple UF2 files. I assume the guide's reference to "the release artifacts located at the bottom of the card" refers to the file "gss-V1_0_R00-v0.1.0.uf2", but I thought I should verify that rather than risk damaging the processor.
![Screenshot 2024-09-05 140516](https://github.com/user-attachments/assets/74c56646-8a1d-49ae-803a-76fa04913438)

Thank you!

**Comment by barulicm** [comment, marked as answer]
You'll need the file corresponding with the version of the hardware you have. The boards are labelled near the top left mounting hole (where the "top" of the board is the edge next to the RJ45 jack).
Most Gizmo users right now have boards from the large bulk order for this season. You likely have the V1.0 board and should download gss-V1_0_R00-v0.1.0.uf2. You can confirm this by looking for "v01.00" printed on your board.

**Reply by jds-designs** [reply]
Thank you for the quick reply, that really helps me out!
